Detectors/GRP/calibration/src/GRPDCSDPsProcessor.cxx

```diff
@@ -230,55 +277,62 @@ bool GRPDCSDPsProcessor::processLHCIFDPs(const DPCOM& dpcom)
     val = o2::dcs::getValue<double>(dpcom);
   }
 
-  for (int ibeam = 1; ibeam <= 2; ++ibeam) {
-    if (aliasStr.find(fmt::format("{}{}", "LHC_IntensityBeam", ibeam)) != string::npos) {
-      updateVector(dpid, mLHCInfo.mIntensityBeam[ibeam - 1], aliasStr, dpcomdata.get_epoch_time(), val);
+  for (int ibeam = 0; ibeam < GRPLHCInfo::BeamAliases::NBeamAliases; ++ibeam) {
+    if (aliasStr.find(static_cast<std::string>(GRPLHCInfo::beamAliases[ibeam])) != string::npos) {
+      updateVector(dpid, mLHCInfo.mIntensityBeam[ibeam], aliasStr, dpcomdata.get_epoch_time(), val);
       return true;
     }
   }
 
   if (aliasStr.find("BPTX") != string::npos) {
-    if (aliasStr == "BPTX_deltaT_B1_B2") {
+    if (aliasStr == static_cast<std::string>(GRPLHCInfo::bptxAliases[GRPLHCInfo::BPTXAliases::BPTX_deltaT_B1_B2])) {
       updateVector(dpid, mLHCInfo.mBPTXdeltaT, aliasStr, dpcomdata.get_epoch_time(), val);
       return true;
     }
-    if (aliasStr == "BPTX_deltaTRMS_B1_B2") {
+    if (aliasStr == static_cast<std::string>(GRPLHCInfo::bptxAliases[GRPLHCInfo::BPTXAliases::BPTX_deltaTRMS_B1_B2])) {
       updateVector(dpid, mLHCInfo.mBPTXdeltaTRMS, aliasStr, dpcomdata.get_epoch_time(), val);
       return true;
     }
-    for (int ibeam = 1; ibeam <= 2; ++ibeam) {
-      if (aliasStr == fmt::format("{}{}", "BPTX_Phase_B", ibeam)) {
-        updateVector(dpid, mLHCInfo.mBPTXPhase[ibeam - 1], aliasStr, dpcomdata.get_epoch_time(), val);
+    for (int ibeam = 0; ibeam < GRPLHCInfo::BPTXPhaseAliases::NBPTXPhaseAliases; ++ibeam) {
+      if (aliasStr == static_cast<std::string>(GRPLHCInfo::bptxPhaseAliases[ibeam])) {
+        updateVector(dpid, mLHCInfo.mBPTXPhase[ibeam], aliasStr, dpcomdata.get_epoch_time(), val);
         return true;
-      } else if (aliasStr == fmt::format("{}{}", "BPTX_PhaseRMS_B", ibeam)) {
-        updateVector(dpid, mLHCInfo.mBPTXPhaseRMS[ibeam - 1], aliasStr, dpcomdata.get_epoch_time(), val);
+      }
+    }
+    for (int ibeam = 0; ibeam < GRPLHCInfo::BPTXPhaseRMSAliases::NBPTXPhaseRMSAliases; ++ibeam) {
+      if (aliasStr == static_cast<std::string>(GRPLHCInfo::bptxPhaseRMSAliases[ibeam])) {
+        updateVector(dpid, mLHCInfo.mBPTXPhaseRMS[ibeam], aliasStr, dpcomdata.get_epoch_time(), val);
         return true;
-      } else if (aliasStr == fmt::format("{}{}", "BPTX_Phase_Shift_B", ibeam)) {
-        updateVector(dpid, mLHCInfo.mBPTXPhaseShift[ibeam - 1], aliasStr, dpcomdata.get_epoch_time(), val);
+      }
+    }
+    for (int ibeam = 0; ibeam < GRPLHCInfo::BPTXPhaseShiftAliases::NBPTXPhaseShiftAliases; ++ibeam) {
+      if (aliasStr == static_cast<std::string>(GRPLHCInfo::bptxPhaseShiftAliases[ibeam])) {
+        LOG(info) << "aliasStr = " << aliasStr << " alias to check = " << static_cast<std::string>(GRPLHCInfo::bptxPhaseShiftAliases[ibeam]);
```

Comment:
this one is logged even in non-verbose mode, could you just suppress it?  Will merge w/o another set of CIs

Reply:
Ah sorry, leftover, I will fix it.

Reply:
Done